### PR TITLE
DRAFT partitioner check FullyConnected with no bias

### PR DESCRIPTION
**Pull request by seanshpark**
On-going draft to check partitioner of FullyConnected with no bias.

Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #7920

**Comment by seanshpark** [comment]
Current state: partitioning is OK but there is redundant empty `Part_Tanh_FC_nobias.00002_npu.circle` is also dropped.
```
$ ls
Part_Tanh_FC_nobias.00001_npu.circle      Part_Tanh_FC_nobias.conn.ini
Part_Tanh_FC_nobias.00002_npu.circle      Part_Tanh_FC_nobias.conn.json
Part_Tanh_FC_nobias.circle                Part_Tanh_FC_nobias.excnt
Part_Tanh_FC_nobias.circle.input0         Part_Tanh_FC_nobias.part
Part_Tanh_FC_nobias.circle.output0        Part_Tanh_FC_nobias.tflite
Part_Tanh_FC_nobias.circle.output0.shape
```

File: compiler/luci/partition/src/PartitionMerge.cpp [hunk 1]
```diff
@@ -58,6 +58,9 @@ bool is_input_same(const luci::PGroup *pgroup, const luci::PGroups *pgroups)
     //         we need to clone this CircleConst for each graph of the group.
     if (dynamic_cast<const luci::CircleConst *>(input) != nullptr)
       continue;
+    // Skip for OutputExclude
+    if (dynamic_cast<const luci::CircleOutputExclude *>(input) != nullptr)
+      continue;
```
Comment (seanshpark) [inline compiler/luci/partition/src/PartitionMerge.cpp:63]: @chunseoklee , can you check with adding this patch (above 3 lines including the comment) and check your model?

Comment (chunseoklee) [reply compiler/luci/partition/src/PartitionMerge.cpp:63]: It works on my local !

**Comment by seanshpark** [comment]
done

**Pull request closed without merging**